Documentation/api_rfc/v2_acl_api.md

```diff
@@ -0,0 +1,436 @@
+# v2 Auth and Permissions
+
+## etcd Resources 
+There are two types of resources in etcd
+1. user resources: users and roles in the user store
+2. key-value resources: key-value pairs in the key-value store
+3. settings resources: acl setting, auth setting, and dynamic etcd cluster settings (election/heartbeat)
+
+### User
```

Comment:
order then as:

user

role

key-value

setting

and permissions should be in its own section?